resources/users/users.model.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const UserSchema = Schema({
+  first_name: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 50
+  },
+  last_name: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 80
+  },
+  username: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 80
+  },
```

Comment:
Please add `unique: true` so that the user has a unique username in the collection